src/App.jsx

```diff
@@ -14,45 +16,121 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ABC = 'alphabetically';
+
+function getPreparedGoods(goods, sortType, reverse) {
+  let result = [...goods];
+
+  if (sortType) {
+    result.sort((good1, good2) => {
+      switch (true) {
+        case sortType === SORT_FIELD_ABC:
+          return good1.localeCompare(good2);
+
+        case sortType === SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
```

Comment:
you still don't fix everything(look at the switch)
<img width="446" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/70220667/5763fc42-7613-4190-85ba-eea1b4ef0158">